src/cbmc/cbmc_parse_options.h

```diff
@@ -83,6 +83,7 @@ class optionst;
   "(round-to-nearest)(round-to-plus-inf)(round-to-minus-inf)(round-to-zero)" \
   OPT_FLUSH \
   "(localize-faults)" \
+  "(use-abstraction):" \
```

Comment:
??? Do the tests actually work?

Reply:
As the option is shared with goto-instrument, could we have `OPT_` and `HELP_` macros?

Reply:
Yes, the regression tests do work because they are using the option in goto-instrument. Cbmc option is not used, so we have dropped it completely.

Reply:
It still seems to appear in the PR...